PyAngle/Angle.py

```diff
@@ -1,373 +1,279 @@
-            elif self.getMinute() < other.getMinute():
-                return -1
-            else:
-                # degrees and minutes are equal
-                if self.getSecond() - other.getSecond() > self.__ABS_TOL:
-                    return 1  # self > other
-                elif other.getSecond() - self.getSecond() > self.__ABS_TOL:
-                    return -1  # self < other
-        # all are equal
-        return 0
-
-    def __eq__(self, other) -> (bool):
+    def __eq__(self, other: 'Angle'):
         """==
         """
         return self.__cmp__(other) == 0
 
-    def __ne__(self, other) -> (bool):
+    def __ne__(self, other: 'Angle'):
         """!=
         """
         return self.__cmp__(other) != 0
 
-    def __le__(self, other) -> (bool):
-        """\<=
+    def __le__(self, other: 'Angle'):
+        """<=
         """
-        return self.__cmp__(other) < 1  # 0, -1s
+        return self.__cmp__(other) <= 0
 
-    def __lt__(self, other) -> (bool):
-        """\<
+    def __lt__(self, other: 'Angle'):
+        """<
         """
-        return self.__cmp__(other) == -1
+        return self.__cmp__(other) < 0
 
-    def __ge__(self, other) -> (bool):
-        """\>=
+    def __ge__(self, other: 'Angle'):
+        """>=
         """
-        return self.__cmp__(other) > -1  # 0, 1
+        return self.__cmp__(other) >= 0
 
-    def __gt__(self, other) -> (bool):
-        """\>
+    def __gt__(self, other: 'Angle'):
+        """>
         """
-        return self.__cmp__(other) == 1
+        return self.__cmp__(other) > 0
 
-    def __str__(self) -> (str):
+    def __str__(self) -> str:
         """Convert angle into string(" " as interval)
         """
         return "{:d} {:d} {:.2f}".format(
-            self.getDegree(), self.getMinute(), self.getSecond()
+            self.deg_i(), self.min_i(), self.sec_f()
         )
 
-    def toDegrees(self) -> (float):
-        """Convert angle into degrees(only degree(float) without minute & second)
+    def to_deg(self) -> float:
+        """Convert angle into degrees(only deg(float) without minute & sec)
         """
-        return float(
-            1.0 * self.getDegree()
-            + self.getMinute() / 60.0
-            + self.getSecond() / 60.0 / 60.0
-        )
+        return float(self._deg)
 
-    def toRadians(self) -> (float):
+    def to_rad(self) -> float:
         """Convert angle into radians(float)
         """
-        return math.radians(self.toDegrees())
+        return math.radians(self.to_deg())
 
-    def toString(self, form="aaa°bbb′ccc″") -> (str):
-        """Convert angle into string of form(default: `"aaa°bbb′ccc″"`)\n
-        `aaa` is degree, `bbb` is minute, `ccc` is second, `DDD` is only degree(float),
-        `RRR` is radian, `XXX` is horizontal ordinate, `YYY` is vertical ordinate.\n
-        > default: `{:.2f}`\n
-        Eg.\n
-        >>> angle.toString()
+    def to_atan2(self, x=None, y=None) -> (float, float):
```

Comment:
I was waiting for DALAO refactoring this ~~rubbish~~

Reply:
submit an issue, or I will forget it